### PR TITLE
Enable continuous delivery

### PR DESCRIPTION
**Pull request by MarkEWaite**
## [JEP-229](https://github.com/jenkinsci/jep/blob/master/jep/229/README.adoc) - Continuous delivery of components

Uses [incrementals tools instructions](https://github.com/jenkinsci/incrementals-tools#superseding-maven-releases) to deliver each time an enhancement pull request is merged.

## Checklist

- [x] I have read the [CONTRIBUTING](https://github.com/jenkinsci/platformlabeler-plugin/blob/master/CONTRIBUTING.md) doc
- [x] I have referenced the Jira issue related to my changes in one or more commit messages
- [x] Unit tests pass locally with my changes
- [x] I have added documentation as necessary
- [x] No spotbugs warnings were introduced with my changes
- [x] I have interactively tested my changes

## Types of changes

- [x] New feature (non-breaking change which adds functionality)

### CODE REVIEW

**Comment by MarkEWaite** [comment]
@jglick I believe that I've followed the steps correctly as described in [superseding maven releases](https://github.com/jenkinsci/incrementals-tools#superseding-maven-releases).  Would be happy to have a review if time allows.

Repository permission updater pull request https://github.com/jenkins-infra/repository-permissions-updater/pull/1834 submitted.

**Review by jglick — APPROVED** [review]
Looks right. :rocket:

**Comment by jglick** [comment]
BTW maybe orthogonal, but I have found that JEP-229 plays nicely with newer GitHub features: set up a branch protection rule on `master` requiring the `Jenkins` status check to pass and the PR branch to be up to date before PRs can be merged (could add mandatory review if there are multiple maintainers), and also enable auto-merge. Then all you need to do to ship a feature is to file a PR, add a suitable label like `enhancement`, click the button requesting auto-merge, and go have dinner—if all goes well, the update will appear within the hour.

**Pull request merged**
